ransack.gemspec

```diff
@@ -2,6 +2,11 @@
 $:.push File.expand_path("../lib", __FILE__)
 require "ransack/version"
 
+def sqlite3_version
+  return '~> 1.4.1' if ENV['RAILS'].nil? || ENV['RAILS'] == '6-0-stable'
```

Comment:
It would be neater to make this a one-liner:
`!ENV['RAILS'] || ENV['RAILS'] == '6-0-stable' ? '~> 1.4.1' :  '~> 1.3.3'`

Reply:
I will do that. Should it also go into the polyamorous gemspec as well, given there are plans to package that as a gem separately?

Reply:
Yes, doing it for polyamorous would be also great. Could you please then just rebase it to 2 commits, one to fix the polyamorous gemspec and another to fix the ransack one? Thanks!

Reply:
Will do, hopefully by next weekend.